pkg/apiserver/apiserver.go

```diff
@@ -289,12 +289,19 @@ func (s *PorchServer) Run(ctx context.Context) error {
 
 	certStorageDir, found := os.LookupEnv("CERT_STORAGE_DIR")
 	if found && strings.TrimSpace(certStorageDir) != "" {
-		if err := setupWebhooks(ctx, webhookNs, certStorageDir); err != nil {
+		// check if we should use cert manager webhook setup
```

Comment:
I think we can omit the comments here. Only include if the flow is unclear.

Reply:
- noted will remove comments unless strictly necessary